packages/cli/src/config/schema.ts

```diff
@@ -1102,6 +1102,15 @@ export const schema = {
 		},
 	},
 
+	redis: {
+		prefix: {
+			doc: 'Prefix for all n8n related keys (wrap in {} for cluster mode)',
```

Comment:
I'm thinking now that the `{}` are related to the hashing strategy, right? This does not relate directly to the prefix. Is this correct? If so, we can remove this comment.

Reply:
I guess we can remove it (the {} are added automatically if forgotten and if in cluster mode)